src/System.Reflection.Metadata/tests/Metadata/Decoding/SignatureDecoderTests.cs

```diff
@@ -15,6 +15,19 @@ namespace System.Reflection.Metadata.Decoding.Tests
 {
     public partial class SignatureDecoderTests
     {
+        private static readonly string RuntimeAssemblyName =
```

Comment:
Rather than hard coding, why not replace with reading off eg `typeof(string).Assembly.FullName` - does that work?

Reply:
Sometimes I'm sad that I don't come up with such clever ways but then I just tell myself that others have been in the business far longer than me ✌

PS: FullName gives you the full qualified assembly name, we want to use `typeof(string).Assembly.GetName().Name` instead.

Reply:
Ok that doesn't work as in core/uap typeof(string).Assembly resolves to System.Private.Corelib. Also when compiling against netstandard typeof(string).Assembly doesn't resolve to netstandard.dll when running them for a particular framework which is wrong here. So we still need the hardcoded values.

Reply:
Nit: You can use hardcoded `mscorlib` for netfx too for consistency.